test/cmdLineTests/runtimemxbeanTests/src/GetPid.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+import java.lang.management.RuntimeMXBean;
+import java.lang.management.ManagementFactory;
+
+class GetPid {
+
+	public static void main(String[] args) {
+		RuntimeMXBean mb = ManagementFactory.getRuntimeMXBean();
+		long id = mb.getPid();
```

Comment:
Maybe can do a comparison with the process id returned by `com.ibm.oti.vm.VM.getProcessId()` here?

Reply:
What's the setup to ensure this test is compiled and run only by `Java 10` and onward JVMs?

Reply:
The targets in `build.xml` are not run if it's not Java 10.